tests/iDibo_tests.py

```diff
@@ -1,11 +1,24 @@
 from nose.tools import *
-import iDibo
 
-def setup():
-    print "SETUP!"
+import unittest
+from gluon.globals import Request
 
-def teardown():
-    print "TEAR DOWN!"
+db = test_db
 
-def test_basic():
-    print "I RAN!"
\ No newline at end of file
+execfile("applications/iDibo/controllers/default.py", globals())
+db(db.auth_user.id > 0).delete()
+db.commit()
+
+
+class TestRegister(unittest.TestCase):
+    def setup(self):
+        request = Request()
+
+    def test_register_title(self):
+        self.controller.register()
+        my_title = self.controller.register.response.title
+        self.assertEqual(my_title, 'iDibo Registration')
+
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
Please use a test "suite", see https://docs.python.org/2/library/unittest.html#basic-example
This gives you a better control and allows you to run through web2py